autopush-common/src/db/bigtable/bigtable_client/error.rs

```diff
@@ -1,26 +1,76 @@
+use backtrace::Backtrace;
 use thiserror::Error;
 
+use crate::errors::ReportableError;
+
 #[derive(Debug, Error)]
 pub enum BigTableError {
-    #[error("Invalid Row Response: {0}")]
+    #[error("Invalid Row Response")]
     InvalidRowResponse(String),
 
-    #[error("Invalid Chunk: {0}")]
+    #[error("Invalid Chunk")]
     InvalidChunk(String),
 
-    #[error("BigTable read error {0}")]
+    #[error("BigTable read error")]
     Read(String),
 
-    #[error("BigTable write error {0}")]
+    #[error("BigTable write error")]
     Write(String),
 
-    #[error("BigTable Admin Error {0}")]
-    Admin(String),
+    #[error("BigTable Admin Error")]
+    Admin(String, Option<String>),
```

Comment:
Does it make sense for this to be an `Option<grpcio::Error>` instead of String (I guess this applies to the other variants as well)?

Reply:
I was debating that. 

Using the grpcio::Error might give a bit more info, but most of the errors aren't really useful beyond the string. 

Meh, I'll convert them.

Reply:
I'll also note that we basically convert these to strings in the only place that they're consumed, when we add them to `extras`, but it's good hygiene anyway.